manifests/install/archive.pp

```diff
@@ -4,7 +4,15 @@
 #
 # PRIVATE CLASS - do not use directly (use main `zookeeper` class).
 class zookeeper::install::archive inherits zookeeper::install {
-  $filename = "${module_name}-${::zookeeper::archive_version}"
+
+
+  # Apache updated the filename base for archive files in release 3.5.5
+  if versioncmp($::zookeeper::archive_version, '3.5.5') <= 0 {
```

Comment:
shouldn't be `>=` here?

Reply:
@andriisabitov Yes, I agree, the logic seems to be inversed. `versioncmp(a, b)` will return `-1` if version `a` is less than version `b`.  In this case for version `3.4.1` the new filename will be used.